From 7f77af06108ac16895a331bcc4cf24269988c596 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 23 Mar 2024 20:14:49 +0100
Subject: [PATCH] Update style for GNOME 46

The app-well-app class is no longer available. Use overview-tile class instead, and update some styles to match with earlier GNOME Shell versions.
---
 stylesheet.css | 30 +++++++++---------------------
 1 file changed, 9 insertions(+), 21 deletions(-)

diff --git a/stylesheet.css b/stylesheet.css
index d221bef7..d5750ffe 100644
--- a/stylesheet.css
+++ b/stylesheet.css
@@ -27,46 +27,34 @@
 	padding: 0; 
 }
 
-#dashtopanelScrollview .app-well-app .overview-icon,
-.dashtopanelMainPanel .show-apps .overview-icon {
+#dashtopanelScrollview .overview-tile,
+.dashtopanelMainPanel .overview-tile {
 	background: none;
-	border: none;
-	margin: 0;
-	padding: 0;
 }
 
-#dashtopanelScrollview .app-well-app .overview-label {
+#dashtopanelScrollview .overview-tile .overview-label {
 	/* must match TITLE_RIGHT_PADDING in apppicons.js */
 	padding-right: 8px;
+	text-align: left;
 }
 
-#dashtopanelScrollview .app-well-app:hover .overview-icon,
-#dashtopanelScrollview .app-well-app:focus .overview-icon {
-	background: none;
-}
-
-.dashtopanelMainPanel .show-apps:hover .overview-icon,
-#dashtopanelScrollview .app-well-app:hover .dtp-container,
-#dashtopanelScrollview .app-well-app:focus .dtp-container {
+#dashtopanelScrollview .overview-tile:hover .dtp-container,
+#dashtopanelScrollview .overview-tile:focus .dtp-container {
 	background-color: rgba(238, 238, 236, 0.1);
 }
 
-#dashtopanelScrollview .app-well-app:hover .dtp-container.animate-appicon-hover {
+#dashtopanelScrollview .overview-tile:hover .dtp-container.animate-appicon-hover {
 	background: none;
 }
 
-#dashtopanelScrollview .app-well-app:active .dtp-container {
+#dashtopanelScrollview .overview-tile:active .dtp-container {
 	background-color: rgba(238, 238, 236, 0.18);
 }
 
-#dashtopanelScrollview .app-well-app .favorite {
+#dashtopanelScrollview .overview-tile .favorite {
 	background-color: rgba(80, 150, 255, 0.4);
 }
 
-#dashtopanelScrollview .app-well-app-running-dot {
-	margin-bottom: 0;
-}
-
 #dashtopanelTaskbar .scrollview-fade {
     background-gradient-end: rgba(0, 0, 0, 0);
 }
